tests/foreman/ui/test_usergroup.py

```diff
@@ -80,18 +63,18 @@ def test_negative_create_usergroup_with_same_name(self, group_name):
         @Assert: Usergroup cannot be  created with existing name
 
         """
+        group_name = gen_string('alphanumeric')
         with Session(self.browser) as session:
-            make_usergroup(
-                session, org=self.organization.name, name=group_name)
+            make_usergroup(session, org=self.organization.name,
+                           name=group_name)
```

Comment:
For fewer parameters like this, I agree that hanging indents are better than hanging code.  I will fix this. 

But UI has some functions which take 10 arguments or so.  This is a code flaw that the function takes 10+ arguments which needs to be addressed.  Until it is addressed, IMO it is better to use hanging code just for those functions which is fewer lines of code vs. hanging indent.